aicsimageio/writers/ome_zarr_writer.py

```diff
@@ -200,6 +209,12 @@ def write_image(
                 "must be before X, Y, and Z."
             )
 
+        if chunk_dims and len(chunk_dims) != ndims:
```

Comment:
Incredible small edge case no one is likely to ever hit, but just in case :shrug:
```suggestion
        if chunk_dims is not None and len(chunk_dims) != ndims:
```